libs/primitives/src/lib.rs

```diff
@@ -39,9 +39,12 @@ pub mod types {
 	/// PoolId type we use.
 	pub type PoolId = u64;
 
-	/// OrderId type we to identify order per epoch.
+	/// OrderId type we use to identify order per epoch.
 	pub type OrderId = u64;
 
+	/// EpochId type we use to identify epochs in our revolving pools
+	pub type EpochId = u32;
```

Comment:
The rewards that @lemunozm and @mikiquantum are working on will also have epochs. Should we separate the types, and call this `PoolEpochId`? Or use the same type for both?

Reply:
Are the reward epochs referring to the pools epochs? If not, I would separate the naming.

Reply:
No they are separate epochs. So let's rename this to `PoolEpochId` then? And @lemunozm maybe you want to use the `RewardEpochId` naming in your PR?

Reply:
Ping @lemunozm

Reply:
I will rename